theano/gof/tests/test_graph_opt_caching.py

```diff
@@ -6,10 +6,8 @@
 import theano
 import theano.tensor as T
 
-
-
 def test_graph_opt_caching():
-    opt_db_file = theano.config.compiledir+'/optimized_graphs.pkl'
+    opt_db_file = theano.config.compiledir+'/*.pkl'
     os.system('rm %s'%opt_db_file)
```

Comment:
We can make a new theano flag for that, that default to the current value.
This flag would be able to change during execution.

On Fri, Jan 16, 2015 at 2:01 PM, abergeron <notifications@github.com> wrote:

> In theano/gof/tests/test_graph_opt_caching.py
> <https://github.com/Theano/Theano/pull/2299#discussion-diff-23112732>:
>
> >  def test_graph_opt_caching():
> > -    opt_db_file = theano.config.compiledir+'/optimized_graphs.pkl'
> > +    opt_db_file = theano.config.compiledir+'/*.pkl'
> >      os.system('rm %s'%opt_db_file)
>
> Can we just change the path where the 'pkl' files are going then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2299/files#r23112732>.
>